typescript/simple-mercari-web/src/components/Listing/Listing.tsx

```diff
@@ -23,44 +27,87 @@ export const Listing: React.FC<Prop> = (props) => {
 
   const onValueChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     setValues({
-      ...values, [event.target.name]: event.target.value,
-    })
+      ...values,
+      [event.target.name]: event.target.value,
+    });
   };
   const onFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     setValues({
-      ...values, [event.target.name]: event.target.files![0],
-    })
+      ...values,
+      [event.target.name]: event.target.files![0],
+    });
   };
   const onSubmit = (event: React.FormEvent<HTMLFormElement>) => {
-    event.preventDefault()
-    const data = new FormData()
-    data.append('name', values.name)
-    data.append('category', values.category)
-    data.append('image', values.image)
+    event.preventDefault();
+    const data = new FormData();
+    data.append("name", values.name);
+    data.append("category", values.category);
+    data.append("image", values.image);
 
-    fetch(server.concat('/items'), {
-      method: 'POST',
-      mode: 'cors',
+    console.log(values);
+    fetch(server.concat("/items"), {
```

Comment:
同じく`await`をつけてあげましょう
```suggestion
    await fetch(server.concat("/items"), {
```

Reply:
丁寧なレビューありがとうございます！反映しました